pre_commit/commands/run.py

```diff
@@ -375,6 +380,7 @@ def run(
             exit_stack.enter_context(staged_files_only(store.directory))
 
         config = load_config(config_file)
+        monitor.set_report_command(config['metrics_command'])
```

Comment:
I don't love this, but the config file is parsed at a kind of awkward time relative to when we want to collect metrics. I think it's okay from a functional perspective, though: it'll just not report metrics if there's an exception before this point, which would mainly stem from problems with the config file, I think.